config-linux.md

```diff
@@ -126,6 +127,12 @@ Each entry has the following structure:
 
 The same `type`, `major` and `minor` SHOULD NOT be used for multiple devices.
 
+Containers MAY NOT access any device node that is not explicitly referenced in
+the **`devices`** array. Rationale: runtimes based on virtual machines need to
```

Comment:
Conflicts with the configLinuxDefaultDevices section

Reply:
@AkihiroSuda Good point. Reworded to indicate this.